introduction.md

```diff
@@ -1,4 +1,9 @@
 Hello, I'm currently on the "Mastering Github" course on Codeschool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?